REQUIRE

```diff
@@ -1,5 +1,6 @@
 julia 0.4
-DataArrays 0.3.4
+NullableArrays 0.0.5
+CategoricalArrays 0.0.2
```

Comment:
What's the error? I tagged these releases a few days ago precisely to make this branch pass the tests.